src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/posthearingreview/PostHearingReviewAboutToSubmitHandlerTest.java

```diff
@@ -95,4 +109,35 @@ void shouldReturnWithoutError() {
 
         assertThat(response.getErrors()).isEmpty();
     }
+
+    @Test
```

Comment:
Convert to Parameterized test. CsvSource for the reviewType and verify expected DocumentType.

Reply:
done